### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by KarolDenst**

### CODE REVIEW

File: Hyper/GameEntities/Humanoid.cs [hunk 1]
```diff
@@ -61,27 +61,33 @@ var collidableReference
 
         LastContactTime = DateTime.UtcNow;
         LastContactBody = collidableReference.BodyHandle;
-#if DEBUG
+
         // TODO replace with something more sensible
```
Comment (AleksyBalazinski) [inline Hyper/GameEntities/Humanoid.cs:65]: this comment can be removed I think

File: Hyper/Menu/Controls.cs [hunk 2]
```diff
@@ -22,6 +22,8 @@ public class Controls : SingleChildWidget
         {"Left Mouse", "Use Item"},
         {"Right Mouse", "Alt Use Item"},
         {"Esc", "Show/Hide Menu"},
+        {"Tab", "Switch camera"},
```
Comment (AleksyBalazinski) [inline Hyper/Menu/Controls.cs:25]: These don't do anything yet right?

File: Hyper/Hyper.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
+    <OutputType>WinExe</OutputType>
```
Comment (AleksyBalazinski) [inline Hyper/Hyper.csproj:4]: This hides console in Debug for me. I'll try to fix this

File: Hyper/Menu/Controls.cs (hunk 2 above)
Comment (KarolDenst) [reply Hyper/Menu/Controls.cs:25]: These show up on the Controls screen

**Review by AleksyBalazinski — APPROVED** [review]

**Review by AleksyBalazinski — COMMENTED** [review]
Maybe change the commit message to something more specific

**Pull request merged**